### PR TITLE
Add the chart of pulsar resources operator

### PR DESCRIPTION
**Pull request by FushuWang**
### Motivation

Add the chart of pulsar resources operator

### Modifications

Add the chart of pulsar resources operator

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### Documentation

Check the box below.

Need to update docs? 

- [x] `doc`

### CODE REVIEW

**Review by fantapsody — APPROVED** [review]

**Review by MarvinCai — APPROVED** [review]

File: charts/pulsar-resources-operator/tests/deployment_test.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+suite: test deployment
```
Comment (maxsxu) [inline charts/pulsar-resources-operator/tests/deployment_test.yaml:1]: Missing license header.
Or skip license test in the `license_test.go`

File: charts/pulsar-resources-operator/tests/serviceaccount_test.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+suite: test service account
```
Comment (maxsxu) [inline charts/pulsar-resources-operator/tests/serviceaccount_test.yaml:1]: Missing license header.
Or skip license test in the `license_test.go`

**Review by maxsxu — CHANGES_REQUESTED** [review]
Left a comment due to the failed CI

**Comment by ericsyh** [comment]
@FushuWang @fantapsody Just check with you will we put the resource operator charts under `streamnative/charts` repo or `streamnative/pulsar-resource-operator` repo?

**Comment by fantapsody** [comment]
> @FushuWang @fantapsody Just check with you will we put the resource operator charts under `streamnative/charts` repo or `streamnative/pulsar-resource-operator` repo?

It will be in `streamnative/pulsar-resource-operator`, I think we can close this one? @FushuWang

**Comment by FushuWang** [comment]
> > @FushuWang @fantapsody Just check with you will we put the resource operator charts under `streamnative/charts` repo or `streamnative/pulsar-resource-operator` repo?
> 
> It will be in `streamnative/pulsar-resource-operator`, I think we can close this one? @FushuWang

Yeah, I will close it @ericsyh @fantapsody

**Pull request closed without merging**